frontend/assets/style/_transitions.scss

```diff
@@ -67,15 +68,15 @@
 .slide-next-enter {
   @include tablet {
     opacity: 0;
-    transform: translate3d(0, -100%, 0);
+    transform: translate3d(0, 100%, 0);
```

Comment:
Note: The animation between settings tabs were reversed. Fixed.

Reply:
i'll work to fix all transitions between elements this week and look at it. It seems weird that this was reversed initially.

Reply:
Okay! If your transition PR is merged before mine, I'll take closer attention to this part